Packs/CortexAttackSurfaceManagement/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Cortex Attack Surface Management",
     "description": "Content for working with Attack Surface Management (ASM).",
     "support": "xsoar",
-    "currentVersion": "1.7.48",
+    "currentVersion": "1.7.49",
```

Comment:
```suggestion
    "currentVersion": "1.7.51",
```

Reply:
The comment is probably a mistake
the current version is 1.7.48
I continue with the merger